src/components/MultiHopTrade/components/TradeConfirm/TradeConfirm.tsx

```diff
@@ -0,0 +1,216 @@
+import { StarIcon } from '@chakra-ui/icons'
+import {
+  Box,
+  Button,
+  Divider,
+  HStack,
+  Spinner,
+  Stack,
+  Step,
+  StepDescription,
+  StepIcon,
+  StepIndicator,
+  StepNumber,
+  Stepper,
+  StepSeparator,
+  StepStatus,
+  StepTitle,
+  useColorModeValue,
+} from '@chakra-ui/react'
+import { useMemo } from 'react'
+import { AssetIcon } from 'components/AssetIcon'
+import { Card } from 'components/Card/Card'
+import { LazyLoadAvatar } from 'components/LazyLoadAvatar'
+import { SlideTransition } from 'components/SlideTransition'
+import { RawText, Text } from 'components/Text'
+import { WithBackButton } from 'components/Trade/WithBackButton'
+import { localAssetData } from 'lib/asset-service'
+
+import LiFiIcon from '../TradeQuotes/lifi-icon.png'
+
+const getIsHopComplete = (_tradeExecutionStatus: unknown, isFirstHop: boolean) => {
+  // TODO: mock function for now
+  return isFirstHop
+}
+
+const getActiveStepperStep = (_tradeExecutionStatus: unknown, isFirstHop: boolean) => {
+  // TODO: mock function for now
+  // Infinity means "render all steps as completed"
+  return isFirstHop ? Infinity : 2
+}
+
+const Hop = ({
+  isApprovalRequired,
+  shouldRenderDonation,
+  isFirstHop,
+}: {
+  isApprovalRequired: boolean
+  shouldRenderDonation: boolean
+  isFirstHop: boolean
+}) => {
+  const backgroundColor = useColorModeValue('gray.100', 'gray.750')
+  const borderColor = useColorModeValue('gray.50', 'gray.650')
+
+  const tradeExecutionStatus = 'mock_status_123'
+
+  const isComplete = getIsHopComplete(tradeExecutionStatus, isFirstHop)
+  const activeStep = getActiveStepperStep(tradeExecutionStatus, isFirstHop)
+
+  const steps: {
+    title: string
+    description?: string
+    stepIndicator: JSX.Element
+    content?: JSX.Element
+  }[] = useMemo(() => {
+    const steps = []
```

Comment:
`const`s are block-scoped and theoretically fine, but we usually try to avoid potential shadowing as a paranoia